Packs/Recorded_Future/Integrations/Recorded_Future/Recorded_Future.yml

```diff
@@ -1,95 +1,109 @@
+category: Data Enrichment & Threat Intelligence
```

Comment:
I see a lot by I believe that most of the things here are not really changes. Right? Can you summarize the changes made here in another way?